test/create-user.test.ts

```diff
@@ -0,0 +1,154 @@
+import "mocha";
+import "reflect-metadata";
+import { expect } from "chai";
+import { createHash } from "crypto";
+import { UserRepository } from "../src/data-source";
+import { getQuery, getMutation } from "./utils";
+import { getToken } from "../src/token";
+
+describe("create user test", () => {
+  //basic input object for testing
+  let input = {
+    profession: "teste",
+    password: "Teste123",
+    name: "teste1",
+    email: "teste@teste.com",
+    dateOfBirth: "01/01/2000",
+  };
```

Comment:
aqui poderia só tipar o input sem inicializar
```suggestion
  let input: InputType;
```